GravatarApp.xcodeproj/project.pbxproj

```diff
@@ -369,6 +369,7 @@
 				GENERATE_INFOPLIST_FILE = YES;
 				INCLUDED_SOURCE_FILE_NAMES = "";
 				INFOPLIST_KEY_NSCameraUsageDescription = "Upload new avatars";
+				INFOPLIST_KEY_NSPhotoLibraryAddUsageDescription = "Save avatars to your Photos app";
```

Comment:
Users may not be that familiar with the "app" word. Some alternatives I could think of:

Save avatars to your Photos
Save to Photos
Save avatars to your photo library